docker/cla_public.ini

```diff
@@ -11,8 +11,9 @@ processes = 2
 chdir = /home/app/flask
 module = cla_public.wsgi
 callable = app
-req-logger = file:/var/log/wsgi/request.log
-logger = file:/var/log/wsgi/error.log
+logger-req=stdio
+log-req-encoder=json {"process_name": "uwsgi", "timestamp": "${strftime:%%Y-%%m-%%dT%%H:%%M:%%S}", ${msg}}
```

Comment:
👏 TIL

💭 Would it be possible to annotate this with milliseconds as well?

Reply:
According to the documentation at https://uwsgi-docs.readthedocs.io/en/latest/LogEncoders.html#core-encoders we could use `${micros}`

> `${micros}` the current unix time in microseconds

E.g. `json {"process_name": "uwsgi", "timestamp_msec": ${micros}, "message": "${msg}"}`

What do you think?

Reply:
Thanks for the tip!